### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by rbogle**
## Changes proposed in this pull request:
- adding changelog for sprint 02-28-22
-
-

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/changelog-02-28)

## Security Considerations
None

### CODE REVIEW

File: _posts/2022-02-18-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+The cloud.gov team is working on providing change logs so everyone can see new features and updates. Highlights this month include changes to several of the platform buildpacks.
+
+# Change Log
+
+## Customer Facing
+
+---
+No changes impacting customers directly this sprint.
```
Comment (apburnes) [inline _posts/2022-02-18-release-notes.md]: We deployed an new version of stratos from v3 to v4

Comment (rbogle) [reply _posts/2022-02-18-release-notes.md]: oops..right!

File: _posts/2022-02-18-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+date: February 18, 2022
```
Comment (mheadd) [inline _posts/2022-02-18-release-notes.md]: I think you'll need to change this to the current date if you want it to display in the Federalist preview. Posts with future dates are displayed.

Comment (rbogle) [reply _posts/2022-02-18-release-notes.md]: well that explains that. :) thx

Comment (mheadd) [reply _posts/2022-02-18-release-notes.md]: Or you can review the preview tomorrow

**Review by apburnes — APPROVED** [review]

**Comment by apburnes** [comment]
Thank you!

**Pull request merged**